tests/e2e-openshift/tls-singletenant/01-assert.yaml

```diff
@@ -104,15 +116,133 @@ spec:
         secret:
           defaultMode: 420
           secretName: tempo-simplest-query-frontend-mtls
+      - name: simplest-ui-oauth-proxy-tls
+        secret:
+          defaultMode: 420
+          secretName: simplest-ui-oauth-proxy-tls
 status:
   availableReplicas: 1
   readyReplicas: 1
   replicas: 1
+
 ---
-apiVersion: apps/v1
-kind: StatefulSet
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  annotations:
+    haproxy.router.openshift.io/timeout: 30s
+  labels:
+    app.kubernetes.io/component: query-frontend
+    app.kubernetes.io/instance: simplest
+    app.kubernetes.io/managed-by: tempo-operator
+    app.kubernetes.io/name: tempo
+  name: tempo-simplest-query-frontend
+spec:
+  port:
+    targetPort: oauth-proxy
+  tls:
+    termination: reencrypt
+  to:
+    kind: Service
+    name: tempo-simplest-query-frontend
+    weight: 100
+  wildcardPolicy: None
+
+---
+apiVersion: v1
```

Comment:
Can we format this file?

Reply:
Its a bug as mentioned in the above comment. https://issues.redhat.com/browse/TRACING-4833 without the distributor tls enabled, the formatting is serialised correctly.